source/ada/lsp-ada_handlers.adb

```diff
@@ -531,6 +531,30 @@ package body LSP.Ada_Handlers is
       LSP.File_Monitors.Unchecked_Free (Self.File_Monitor);
    end Cleanup;
 
+   ----------------
+   -- Clean_Logs --
+   ----------------
+
+   procedure Clean_Logs (Self : access Message_Handler; Dir : Virtual_File) is
+      Log_Files : File_Array_Access := Read_Dir (Dir, Files_Only);
```

Comment:
"Files" might be a better name, since it may not only contain log files

Reply:
Done